src/components/flow/actions/sendinteractivemsg/helpers.tsx

```diff
@@ -195,7 +195,23 @@ export const stateToRouter = (
     valid: true
   };
 
-  const renderedNode = stateToNode(settings, result, assetStore);
+  let renderedNode;
+
+  if (settings.originalNode.ghost) {
+    renderedNode = stateToNode(settings, result, assetStore);
+  } else {
+    settings = {
+      ...settings,
+      originalNode: {
+        ...settings.originalNode,
+        node: {
+          ...settings.originalNode.node,
+          uuid: settings.originalNode.node.exits[0].destination_uuid
```

Comment:
Hey we are assuming here the destination id exists. What happens for the case when there is no destination id?